cluster/gce/cloud-init/master.yaml

```diff
@@ -38,7 +38,8 @@ write_files:
         enable_tls_streaming = true
       [plugins.cri.cni]
         bin_dir = "/home/containerd/opt/cni/bin"
```

Comment:
any reason this still needs to be under ```/home/containerd``` Why not keep it consistent ? :)

Reply:
Because we place all cni binaries there.

If I remember correctly, `/opt/cni/bin` on COS image is readonly. That's why we can't put it there.